app/assets/javascripts/tooltips.js

```diff
@@ -0,0 +1,13 @@
+jQuery(function($) {
+  var tooltipElements = $('.advanced-tooltip');
+  var input = $('input[id^="new_password"]');
```

Comment:
why not use a class? I actually had to look up the selector. also, this implementation is specific to the use case.